manifests/crds/rollout-crd.yaml

```diff
@@ -308,6 +308,8 @@ spec:
                         type: object
                       stableService:
                         type: string
+                      checkNewRSOnly:
+                        type: boolean
```

Comment:
Can we also document this change at : https://github.com/argoproj/argo-rollouts/blob/master/docs/features/canary.md#other-configurable-features? Just to make the doc update to date.

Reply:
Done!

Reply:
Hi, do you know how to pass the codegen verify task after modify rollout crd?